src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -18,11 +17,12 @@ import { ToastrOvenService } from '../../../shared/modules/toaster/notification.
 import { AuthService } from '../../../core/auth-module/auth.service';
 import { Collection } from '../../../core/collection-module/collections.service';
 import { UserService } from 'app/core/user-module/user.service';
+import { SearchService } from 'app/core/learning-object-module/search/search.service';
 @Component({
   selector: 'clark-learning-objects',
   templateUrl: './learning-objects.component.html',
   styleUrls: ['./learning-objects.component.scss'],
-  providers: [PublicLearningObjectService]
+  providers: []
```

Comment:
does there need to be anything in this providers array?

Reply:
Not with the new organizational structure we are doing with the services.